assets/app/actions/dispatchActions.js

```diff
@@ -0,0 +1,72 @@
+import { dispatch } from '../store';
```

Comment:
Cool, that makes sense to me. I actually think that the placement of business logic is, weirdly, not really discussed much in terms of React application architecture, this kind of separation between the composition and dispatching actions seems really helpful!